docs/4.x/heatmap-session-recording/faq.md

```diff
@@ -244,3 +244,29 @@ console of your browser.
 ## How do I solve the issue of a partially rendered page with pictures or content missing?
 
 The screenshot for a Heatmap is taken when the page is officially loaded according to the browser. In some cases, the browser may consider the page fully loaded and rendered before all components are actually loaded.  Learn how to enable heatmaps once the page is fully loaded [in this FAQ](https://matomo.org/faq/heatmap-session-recording/faq_25413/).
+
+
+## How do I disable loading of CSS from database for Heatmap and Session Recording?
+
+Since the release of version **4.3.0** of Heatmap and Session Recording plugin, Matomo now tries to save the CSS internally in database by default to ensure the recording works even after the CSS link expires. However you can disable this feature by setting below setting in your config/config.ini.php.
```

Comment:
```suggestion
Since the release of version **4.3.0** of Heatmap and Session Recording plugin, Matomo now tries to save the CSS internally in database by default to ensure the recording works even after the CSS link expires. However, you can disable this feature by setting below setting in your config/config.ini.php.
```